modules/tables/table_31_by_lakshay.html

```diff
@@ -0,0 +1,45 @@
+<html>
```

Comment:
Everything looks great, just remove the html tag.

1) remove the html tag. Format:
```
<div>

</div>

<style>

</style>

<script>

</script>

```

2) Use Id or class for styling Elements(tags): The reason for this issue is to create a module that can be imported into any project. Some projects(HTML page) may contain same tags (*, div, h1, a, table, p etc) that are inside your module. Styling your tags would mean styling those tags too. So use unique id or classes for your styling.